templates/etc/systemd/system/docker.service.j2

```diff
@@ -2,21 +2,37 @@
 
 [Unit]
 Description=Docker Application Container Engine
-Documentation=http://docs.docker.com
+Documentation=https://docs.docker.com
 After=network.target docker.socket
 Requires=docker.socket
 
 [Service]
-EnvironmentFile=-/etc/default/docker
```

Comment:
This will break Docker from Debian configured by this role. You should only remove it for apropriate Docker versions.